assembly/src/main/dist/docker/Dockerfile.amd64

```diff
@@ -1,30 +1,32 @@
 FROM amd64/tomcat:8.5-jre8-alpine
 MAINTAINER Bart Hanssens (bart.hanssens@bosa.fgov.be)
 
-ARG VERSION="2.5.1"
-
 ENV JAVA_OPTS="-Xmx2g"
 ENV CATALINA_OPTS="-Dorg.eclipse.rdf4j.appdata.basedir=/var/rdf4j"
 
-RUN adduser -S tomcat 
+RUN adduser -S tomcat
+
+COPY ignore/rdf4j.zip /tmp/rdf4j.zip
+
+WORKDIR /tmp
 
-RUN cd /tmp && \
-	wget -q "https://www.eclipse.org/downloads/download.php?file=/rdf4j/eclipse-rdf4j-${VERSION}-sdk.zip&r=1" -O /tmp/rdf4j.zip && \
-	unzip -q /tmp/rdf4j.zip && \
-	rm -rf /usr/local/tomcat/webapps/* && \
-	cp /tmp/eclipse-rdf4j-${VERSION}/war/*.war /usr/local/tomcat/webapps && \
-	rm -rf /tmp/eclipse && \
-	rm /tmp/rdf4j.zip && \
-	mkdir -p /var/rdf4j && \
-	chown -R tomcat /var/rdf4j /usr/local/tomcat && \
-	chmod a+x /usr/local/tomcat /usr/local/tomcat/bin /usr/local/tomcat/bin/catalina.sh
+RUN	unzip -q /tmp/rdf4j.zip
```

Comment:
IIRC, each RUN will add a layer, which will increase the size of the docker image (hence the '&&' hack, especially for not increasing the image size with unzipped files etc that will be eventually removed).

Perhaps the "new" docker multistage build feature https://docs.docker.com/develop/develop-images/multistage-build/ would be a better approach ?

Reply:
Multistage isn't worth it unless we are doing the actual maven building inside the docker image. Single layer for this makes sense though.